locales/locales/en.json

```diff
@@ -1629,6 +1629,27 @@
 	"accountSwitcherReloadOtherTabsDesc": {
 		"message": "After switching accounts, automatically reload other tabs."
 	},
+	"accountSwitcherAutoSwitchOnPostTitle": {
+		"message": "Automatically Switch To Post OP"
+	},
+	"accountSwitcherAutoSwitchOnPostDesc": {
+		"message": "Automatically switch to alternative account if it's the OP of the post you are on."
+	},
+	"accountSwitcherAutoSwitchOnCommentTitle": {
+		"message": "Automatically Switch To Comment OP"
+	},
+	"accountSwitcherAutoSwitchOnCommentDesc": {
+		"message": "Automatically switch to alternative account if a comment from one of your alternative accounts is on the page"
```

Comment:
How about ...

"Automatically switch to account in Account Switcher if that account has submitted a comment to the current post."